mod/gc_elgg_sitemap/start.php

```diff
@@ -76,6 +77,21 @@ function gc_elgg_sitemap_init() {
 	}
 }
 
+function elgg_display_comment_handler($hook, $type, $return, $params) {
+    $segments = elgg_extract('segments', $return, array());
+
+    if ($segments[0] === 'view') {
+        $entity = get_entity($segments[1]);
+        $title = $entity->description;
+        $content = $entity->description;
+
+        echo elgg_view_page($title, $content);
```

Comment:
$entity->description twice?

Reply:
will remove the description for the title